### PR TITLE
Bump all dependency versions

### PR DESCRIPTION
**Pull request by IRDonch**

### CODE REVIEW

**Comment by IRDonch** [comment]
Jenkins please retry a build

File: ci/requirements-ac-test.txt [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
 editdistance==0.5.3
-importlib-metadata==1.5.0  # via pluggy, pytest
+importlib-metadata==1.6.0  # via pluggy, pytest
 joblib==0.14.1            # via scikit-learn
 more-itertools==8.2.0     # via pytest
-nibabel==3.0.1
-numpy==1.17.5             # via nibabel, scikit-learn, scipy
-packaging==20.1           # via pytest
+nibabel==3.0.2
+numpy==1.18.2             # via nibabel, scikit-learn, scipy
```
Comment (eaidova) [inline ci/requirements-ac-test.txt:8]: looks like I need to update update_dependencies script...

Comment (IRDonch) [reply ci/requirements-ac-test.txt:8]: What do you mean?

Comment (eaidova) [reply ci/requirements-ac-test.txt:8]: this https://github.com/opencv/open_model_zoo/pull/1039/files

Comment (eaidova) [reply ci/requirements-ac-test.txt:8]: we still preserve old numpy due to pycocotools win fork... But dependencies were divided to core and extra for support ARM

**Comment by IRDonch** [comment]
Jenkins please retry a build

**Comment by IRDonch** [comment]
Jenkins please retry a build

**Pull request merged**